examples/k8s/deployment.yaml

```diff
@@ -113,10 +116,10 @@ webhooks:
       port: 8443
       namespace: "devops"
   sideEffects: None
-  admissionReviewVersions: ["v1beta1"]
+  admissionReviewVersions: ["v1", "v1beta1"]
   failurePolicy: Ignore
-  name: v1beta1.kanopy-platform.github.io
-  reinvocationPolicy: IfNeeded
+  name: hedgetrimmer.kanopy-platform.github.io
+  reinvocationPolicy: Never
```

Comment:
During initial setup I copied pasted `IfNeeded` here. Looking at [ReinvocationPolicy](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#reinvocation-policy), `Never` should be ok for us and is the default. I tested this setting for a few resources and it works ok.

Reply:
We need reinvocation for things like sidecar injection.

Reply:
Oh, would a possible flow be like we get it first, mutate. Then gets to istio, which injects sidecar, but it'll need to come back to us to add the resources requirements to the sidecar?

Reply:
Yeah. hedgetrimmer cares about all the container and initcontainer definitions and the sidecar mutatingwebhook would inject another container. That only happens at the pod level but over all we care if something else changes the list of containers or initcontainers on any of of the workload types.

Reply:
Gotcha, reverted to `IfNeeded` and will change the kanopy module to match